packages/react-components/react-datepicker-compat/stories/DatePicker/DatePickerDateBoundaries.stories.tsx

```diff
@@ -8,7 +8,7 @@ const useStyles = makeStyles({
   },
 });
 
-const today = new Date(Date.now());
+const today = new Date('05/24/2023');
```

Comment:
You need to modify also date formatting to make it deterministic i.e.:

(also in `DatePickerErrorHandling.stories.tsx`)

```ts
const today = new Date('05/24/2023');
const minDate = addMonths(today, -1);
const maxDate = addYears(today, 1);

function formatDate(date: Date) {
  return `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()}`;
}
```

Why? Locales on a server and client might be different like in my case:

```
"The date boundaries for this example are 5/7/2023 to 6/7/2024." Client: "The date boundaries for this example are 07/05/2023 to 07/06/2024."
```